src/mqt/bench/targets/gatesets/rigetti.py

```diff
@@ -0,0 +1,157 @@
+        super().__init__("rxpi", 1, [np.pi], label=label)
+
+    def _define(self) -> None:
+        """Define RX(π) gate using standard RX."""
+        q = QuantumRegister(1, "q")
+        qc = QuantumCircuit(q)
+        qc.rx(np.pi, 0)
+        self.definition = qc
+
+
+class RXPI2Gate(Gate):  # type: ignore[misc]
+    r"""Single-qubit RX(π/2) gate.
+
+    **Circuit symbol:**
+    .. parsed-literal::
+             ┌──────────┐
+        q_0: ┤ RX(π/2)  ├
+             └──────────┘
+
+    **Matrix representation:**
+
+    .. math::
+
+        RX(π/2) =
+            \begin{pmatrix}
+                \cos(π/4) & -i \sin(π/4) \\
+                -i \sin(π/4) & \cos(π/4)
+            \end{pmatrix}
+    """
+
+    def __init__(self, label: str | None = None) -> None:
+        """Create RX(π/2) gate."""
+        super().__init__("rxpi2", 1, [np.pi / 2], label=label)
+
+    def _define(self) -> None:
+        """Define RX(π/2) gate using standard RX."""
+        q = QuantumRegister(1, "q")
+        qc = QuantumCircuit(q)
+        qc.rx(np.pi / 2, 0)
+        self.definition = qc
+
+
+class RXPI2DgGate(Gate):  # type: ignore[misc]
+    r"""Single-qubit RX(-π/2) gate.
+
+    **Circuit symbol:**
+    .. parsed-literal::
+             ┌────────────┐
+        q_0: ┤ RX(-π/2)   ├
+             └────────────┘
+
+    **Matrix representation:**
+
+    .. math::
+
+        RX(-π/2) =
+            \begin{pmatrix}
+                \cos(π/4) & i \sin(π/4) \\
+                i \sin(π/4) & \cos(π/4)
+            \end{pmatrix}
+    """
+
+    def __init__(self, label: str | None = None) -> None:
+        """Create RX(-π/2) gate."""
+        super().__init__("rxpi2dg", 1, [-np.pi / 2], label=label)
+
+    def _define(self) -> None:
+        """Define RX(-π/2) gate using standard RX."""
+        q = QuantumRegister(1, "q")
+        qc = QuantumCircuit(q)
+        qc.rx(-np.pi / 2, 0)
+        self.definition = qc
+
+
+def u_gate_equivalence(sel: EquivalenceLibrary) -> None:
+    """Add U(θ, φ, λ) gate equivalence to the SessionEquivalenceLibrary using RZ and RX gates."""
+    theta = Parameter("θ")
+    phi = Parameter("φ")
+    lam = Parameter("λ")
+
+    qr = QuantumRegister(1, "q")
+    qc = QuantumCircuit(qr)
+
+    # Decomposition: U(θ, φ, λ) = RZ(φ) RX(-π/2) RZ(θ) RX(π/2) RZ(λ)
+    qc.append(RZGate(phi), [qr[0]])
+    qc.append(RXPI2DgGate(), [qr[0]])
+    qc.append(RZGate(theta), [qr[0]])
+    qc.append(RXPI2Gate(), [qr[0]])
+    qc.append(RZGate(lam), [qr[0]])
+
+    sel.add_equivalence(UGate(theta, phi, lam), qc)
+
+
+def rx_gate_equivalence(sel: EquivalenceLibrary) -> None:
+    """Add RX(θ) gate equivalence using RX(±π/2) and RZ gates."""
+    theta = Parameter("θ")
+    qr = QuantumRegister(1, "q")
+    qc = QuantumCircuit(qr)
+
+    qc.rz(-np.pi / 2, qr[0])
+    qc.append(RXPI2Gate(), [qr[0]])
+    qc.rz(theta, qr[0])
+    qc.append(RXPI2DgGate(), [qr[0]])
+    qc.rz(np.pi / 2, qr[0])
+
+    sel.add_equivalence(RXGate(theta), qc)
```

Comment:
This doesn't really help much I suppose.
What I actually meant with my other comment was to add equivalences from 
`RXGate(pi/2)` to `RXPI2Gate` and so on.

Reply:
Is this what you would like to see here?
```python
def rx_gate_equivalences(sel: EquivalenceLibrary) -> None:
    """Add RX(θ) gate equivalence using RX(±π/2) and RZ gates."""
    qr = QuantumRegister(1, "q")
    qc = QuantumCircuit(qr)
    qc.append(RXPI2Gate(), [qr[0]])
    sel.add_equivalence(RXGate(np.pi / 2), qc)

    qr = QuantumRegister(1, "q")
    qc = QuantumCircuit(qr)
    qc.append(RXPI2DgGate(), [qr[0]])
    sel.add_equivalence(RXGate(-np.pi / 2), qc)

    qr = QuantumRegister(1, "q")
    qc = QuantumCircuit(qr)
    qc.append(RXPIGate(), [qr[0]])
    sel.add_equivalence(RXGate(np.pi), qc)
```

If yes: Apparently it is not possible to add these equivalences only for specific angles.

Reply:
Yeah. Something like that is what I would have expected.
How's that not supported? Or what is the error you are seeing?

Reply:
It leads to an internal Qiskit Error in `qiskit.transpiler.passes.basis.basis_translator.BasisTranslator.run`: `AttributeError: 'float' object has no attribute '_uuid'` because for a parameterized gate, Qiskit apparently expects a parameter instead of just a specific angle.

Reply:
Hm.. unfortunate..
That's something that I would consider opening up an issue on the Qiskit tracker for.
Such equivalences could be extremely helpful.

Any how.. I suppose this PR can be merged then 😌